services/app-api/src/emailer/emails/sendUndoWithdrawnRateStateEmail.ts

```diff
@@ -0,0 +1,222 @@
+    }
+
+    //This gets us the rate name and summary URL for the rate
+    const rateName =
+        rate.packageSubmissions[0].rateRevision.formData.rateCertificationName
+    const rateID = rate.packageSubmissions[0].rateRevision.formData.rateID
+    if (!rateName || !rateID) {
+        return new Error(
+            `Error parsing for ${rateID ? 'rateCertificationName' : 'rateID'}`
+        )
+    }
+
+    const rateInfo: rateInfo = {
+        rateName,
+        rateURL: rateSummaryURL(rateID, config.baseUrl),
+    }
+
+    //Gathering contracts and state contacts associated to rate
+    const contractRevisions = rate.packageSubmissions[0].contractRevisions
+    if (contractRevisions.length === 0) {
+        return new Error('Rate does not have any contract revisions')
+    }
+    const parsedContractsAndContacts = parseContractsAndContacts(
+        contractRevisions,
+        statePrograms,
+        config
+    )
+
+    if (parsedContractsAndContacts instanceof Error) {
+        return parsedContractsAndContacts
+    }
+
+    const { associatedContracts, stateContactEmails } =
+        parsedContractsAndContacts
+
+    return {
+        latestStatusAction,
+        rateInfo,
+        associatedContracts,
+        stateContactEmails,
+    }
+}
+
+export const sendUndoWithdrawnRateStateEmail = async (
+    rate: RateType,
+    statePrograms: ProgramType[],
+    config: EmailConfiguration
+): Promise<EmailData | Error> => {
+    const undoWithdrawnRateData = validateAndParseUnwithdrawnRate(
+        rate,
+        statePrograms,
+        config
+    )
+
+    if (undoWithdrawnRateData instanceof Error) {
+        return undoWithdrawnRateData
+    }
+
+    const {
+        latestStatusAction, //Contains update information
+        rateInfo, //Rate name and summary URL
+        associatedContracts, //Contracts connected to the rate
+        stateContactEmails,
+    } = undoWithdrawnRateData
+
+    const toAddresses = pruneDuplicateEmails([
+        ...stateContactEmails,
+        ...config.devReviewTeamEmails,
+    ])
+
+    const etaData: undoWithdrawnRateEtaData = {
+        rateInfo: rateInfo,
+        updatedBy: latestStatusAction.updatedBy.email,
+        updatedOn: formatCalendarDate(
+            latestStatusAction.updatedAt,
+            'America/Los_Angeles'
+        ),
+        reason: latestStatusAction.updatedReason,
+        associatedContracts: associatedContracts,
+    }
+
+    const template = await renderTemplate<undoWithdrawnRateEtaData>(
+        'sendUndoWithdrawnRateStateEmail',
+        etaData
+    )
+
+    if (template instanceof Error) {
+        return template
+    } else {
+        return {
+            toAddresses,
+            replyToAddresses: [],
+            sourceEmail: config.emailSource,
+            subject: `${
+                config.stage !== 'prod' ? `[${config.stage}] ` : ''
+            }${etaData.rateInfo.rateName} was unwithdrawn`,
```

Comment:
This subject should be
```suggestion
            }${etaData.rateInfo.rateName} status update`,
```